sdk/identity/azure-identity/test/ut/token_credential_test.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "credential_base_test.hpp"
```

Comment:
This header is only used in this one cpp, should we just move all the declarations from there to this file, and remove the header?